linode/manager
# refactor: [M3-7582] - Remove Enzyme

## Description 📝

Removing Enzyme in favor or React Testing Library has been in the works for years now. This PR's goal is to finally get Enzyme out of here 🗑️. We're really close, so I'm just going to do it all in one sweep with this PR. 

Also, removing Enzyme will make updating to React 18 a bit easier.

## How to test 🧪

- Verify unit tests pass in Github Actions
- Optional: check out this PR and run some of the unit tests that have been touched
- Verify that we're not losing any meaningful test coverage 

## As an Author I have considered 🤔
